openlibrary/templates/subjects.html

```diff
@@ -35,12 +30,7 @@ <h1 class="inline">
             <strong><span><a href="/search?$page.subject_type=$page.name.replace('&','%26')" title="$_('See all works')">$ungettext("%(count)d work", "%(count)d works", page.work_count, count=page.work_count)</a></span></strong>
         </span>
     </span>
-    $if has_tag:
-      <h3 itemprop="description">
-        $page.tag['description']
-      </h3>
```

Comment:
I think that we do want this somewhere on the page, just not in a heading element (and maybe somewhere else on the page).  Maybe displaying this before `tag.body` in `/type/tag/view.html` is the right spot?  This would ensure that the description is always present in the view, but prevents it from appearing on the page twice when `tag.body` is a rendered subject page.